js/adapt-languagePicker.js

```diff
@@ -60,10 +60,16 @@ define([
     }
     
     function setupNavigationView () {
+        var courseGlobals = Adapt.course.get('_globals')._extensions;
+        var navigationBarLabel = 'Select course language';
```

Comment:
Hard-coded strings aren't a good idea. Could you make the ```"aria-label"``` a null string if it's not available in the _globals object?
I'm not entirely sure why it wouldn't be available. It comes through from the schema and the AT. What's the use-case for this?